src/js/angular/core/services/repositories.service.js

```diff
@@ -110,31 +130,33 @@ repositories.service('$repositories', ['$http', 'toastr', '$rootScope', '$timeou
         };
 
         this.init = function (successCallback, errorCallback, quick) {
+            this.loading = true;
             if (!quick) {
                 this.locationsShouldReload = true;
             }
-            this.loading = true;
-            // noCancelOnRouteChange Prevent angularCancelOnNavigateModule.js from canceling this request on route change
             LocationsRestService.getActiveLocation().then(
                 function (res) {
                     if (res.data) {
                         const location = res.data;
                         if (location.active) {
-                            RepositoriesRestService.getRepositories().then(function (res) {
+                            RepositoriesRestService.getRepositories().then(function (result) {
                                     that.location = location;
-                                    that.repositories = res.data;
+                                    mapValues(result.data, (value, key) => {
```

Comment:
You can use `Object.entries(result.data).forEach((key, value) => {...})` here, as you don't use the result from the `mapValues` fn.